tests/DevBetterWeb.Tests/Services/AlumniGraduationServiceTests/CheckIfAnyMemberGraduating.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Core.ValueObjects;
+using DevBetterWeb.Infrastructure.Identity.Data;
+using DevBetterWeb.Infrastructure.Services;
+using Microsoft.AspNetCore.Identity;
+using Moq;
+using Xunit;
+
+namespace DevBetterWeb.Tests.Services.AlumniGraduationServiceTests
+{
+  public class CheckIfAnyMemberGraduating
+  {
+    private readonly Mock<IUserLookupService> userLookupService;
+    private readonly Mock<IRepository<Member>> repository;
+    private readonly Mock<IGraduationCommunicationsService> graduationCommunications;
+    private readonly Mock<UserManager<ApplicationUser>> userManager;
+
+    private const int DAYS_IN_TWO_YEARS = 365 * 2;
+
+    private AlumniGraduationService service;
+
+    public CheckIfAnyMemberGraduating()
+    {
+      userLookupService = new Mock<IUserLookupService>();
+      repository = new Mock<IRepository<Member>>();
+      graduationCommunications = new Mock<IGraduationCommunicationsService>();
+      var store = new Mock<IUserStore<ApplicationUser>>();
+      userManager = new Mock<UserManager<ApplicationUser>>(store.Object, null, null, null, null, null, null, null, null);
```

Comment:
this big interface you have to mock is a code smell. Would it be possible to define a smaller interface that only had the method(s) you need from UserManager, and mock that instead? Then you'd have an adapter somewhere that implemented the small interface with Usermanager, but this code wouldn't care about that.